<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_1</class>
 <widget class="QDialog" name="login_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background:white;
color:black</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_username">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 24px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:24px</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; font-weight:600;&quot;&gt;USERNAME&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; font-weight:600;&quot;&gt;PASSWORD&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_login">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>501</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_login{color:white;
background-color:red;
border-radius:10px;
font-size: 32px;
font : bold;}

#pushButton_login:hover{
background-color:#990000;}</string>
   </property>
   <property name="text">
    <string>LOGIN</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_register">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>400</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
background-color:white;
border-radius:10px;

</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>731</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#cc0000</string>
   </property>
   <widget class="QLabel" name="label_picture">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>211</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>391</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:600;&quot;&gt;WELCOME TO&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:600;&quot;&gt;T&amp;amp;T TICKET MASTER&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton_login</tabstop>
  <tabstop>pushButton_register</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>lineEdit_username</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
